D1673/P1673.md

```diff
@@ -6012,6 +6014,8 @@ constexpr auto scaled(
 [2]{.pnum} *Effects:* Equivalent to
 `return mdspan{x.data(), x.mapping(), accessor_scaled{alpha, x.accessor()}};`.
 
+(TZL) TODO: The non-collapsing behavior should be normative, and the rest of
```

Comment:
@tzlaine Thanks for pointing this out!  Do you think the non-collapsing behavior would be sufficiently implied in the "Effects: Equivalent to" wording, or do we need more normative wording to require this?

Reply:
Yes, it would -- good point.  "Equivalent to:" really means "exactly this code:".